Src/AutoFixture/Kernel/FieldSpecification.cs

```diff
@@ -95,7 +95,9 @@ private bool FieldIsCompatibleWithTargetType(object request)
 
         private bool FieldHasTargetName(object request)
         {
-            return ((FieldInfo)request).Name == this.targetName;
+            return ((FieldInfo)request)
+                   .Name
+                   .Equals(this.targetName, StringComparison.OrdinalIgnoreCase);
```

Comment:
May I suggest that you put this particular pull request aside for a little while, and submit another pull request that expands on the comparison options for the affected Specifications in the kernel?

That would enable us to have a discussion about that, decoupled from the `[Freeze]` attribute.